README-ko.md

```diff
@@ -0,0 +1,446 @@
+<a href="http://www.perfumejs.com/">
+  <img src="https://github.com/Zizzamia/perfume.js/blob/master/docs/src/assets/perfume-logo-v5-0-0.png"
+  align="left" width="200" alt="Perfume.js logo" />
+</a>
+
+# [Perfume.js v5.3.0](http://perfumejs.com)
+
+[![Current version](https://img.shields.io/github/tag/zizzamia/perfume.js?color=3498DB&label=version)](https://www.npmjs.org/package/perfume.js) [![Test Coverage](https://api.codeclimate.com/v1/badges/f813d2f45b274d93b8c5/test_coverage)](https://codeclimate.com/github/Zizzamia/perfume.js/test_coverage) <img alt="No dependencies" src="https://img.shields.io/badge/dependencies-none-27ae60.svg"> [![Build Status](https://travis-ci.org/Zizzamia/perfume.js.svg?branch=master)](https://travis-ci.org/Zizzamia/perfume.js) [![NPM Downloads](http://img.shields.io/npm/dm/perfume.js.svg)](https://www.npmjs.org/package/perfume.js) [![gzip size](https://img.badgesize.io/https://unpkg.com/perfume.js?compression=gzip&label=JS+gzip+size)](https://unpkg.com/perfume.js) [![brotli size](https://img.badgesize.io/https://unpkg.com/perfume.js?compression=brotli&label=JS+brotli+size)](https://unpkg.com/perfume.js)
+
+> <b>페이지 속도</b> 는 기능이며, 이를 제공하려면 작용하는 많은 요인과 근본적인 제한을 이해해야 합니다. 측정 할 수 있다면 개선 할 수 있습니다.
+
+<br />
+<br />
+<br />
+<br />
+
+[English](./README.md)| [简体中文](./README-zh_CN.md) | [Italian](./README-it.md) | 한국어
+
+## 왜 Perfume.js인가?
+
+Perfume은 필드 데이터를 즐겨 찾는 분석 도구로, 다시 보고자 하는 웹 퍼포먼스를 모니터링 할 수 있는 작은 라이브러리입니다. 
```

Comment:
i think this might sound a bit more natural @Zizzamia 
> Perfume은 웹 퍼포먼스를 모니터링 하여 즐겨 찾는 분석 도구로 필드 데이터를 리포트할 수 있는 경량 라이브러리입니다. 

![image](https://user-images.githubusercontent.com/966662/100156204-5e11e780-2e5d-11eb-9245-7ff1499d9ede.png)

Reply:
Okay! I will pr again after modifying that part. Thank you!

Reply:
@ire4564 can you introduce @bangtoven suggestion in this PR please.

Reply:
@Zizzamia In this PR He asked me to change the sentence structure to make it a little smoother from the previous sentence I wrote! So I modified it.